### PR TITLE
testing/py-ostinato: new aport

### PR DESCRIPTION
**Pull request by little-dude**
Decription: python agent for the ostinato controller
Project URL: http://ostinato.org/
Github URL: https://github.com/pstavirs/ostinato/

### CODE REVIEW

File: testing/py-ostinato/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgrel=0
+pkgdesc="A python agent for the ostinato controller"
+url="http://ostinato.org/"
+arch="noarch"
+license="GPLv3"
+depends="python"
+makedepends="python-dev py-setuptools"
+source="http://pypi.python.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_pkgname-$pkgver
```
Comment (jirutka) [inline testing/py-ostinato/APKBUILD]: Please rename `_builddir` to `builddir` (this has been changed in abuild tool recently).

File: testing/py-ostinato/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-ostinato
+_pkgname=python-ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="A python agent for the ostinato controller"
+url="http://ostinato.org/"
+arch="noarch"
+license="GPLv3"
+depends="python"
```
Comment (jirutka) [inline testing/py-ostinato/APKBUILD]: According to `setup.py` included in `python-ostinato-0.7.1.tar.gz`, this module requires `protobuf>=2.3.0` — corresponding Alpine package must be specified in `depends`.

Comment (little-dude) [inline testing/py-ostinato/APKBUILD]: oh ok. I thought for python packages we didn't need to specify the dependencies, sorry.

**Comment by jirutka** [comment]
You’re author of this apkbuild, aren’t you? Then you should add yourself as a contributor – `# Contributor: Corentin Henry <corentinhenry@gmail.com>` at the first line. Besides that this PR is okay.

⚠️ Merge this **after** #76.

**Comment by jirutka** [comment]
@little-dude Google did it again… 😠 #116

**Comment by jirutka** [comment]
Merged in 38310f978752fb7a21539e596e169a739a56fe02, thanks!

**Pull request closed without merging**